src/client/components/AdminPanel/Analytics/index.js

```diff
@@ -15,6 +15,12 @@ const Analytics = () => {
     });
   };
 
+  const getExcelFile = () => {
+    ltikPromise.then(ltik => {
+      window.open(`http://localhost:3000/api/analyticsfile?ltik=${ltik}`);
```

Comment:
@finnzink This has a hardcoded full path URL.. This will not work when we deploy to production.

Reply:
@rlorenzo Nice catch. Fixed